src/Circlereader/util/initSubgraphNodes.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {SubGraph} from '../circle-analysis/circle/sub-graph';
+import {Node} from '../type/types';
+
+export function initSubgraphNodes(subgraph: SubGraph): Array<Node> {
+  let nodesArr: Array<Node> = [];
+
+  for (let tensorIdx = 0; tensorIdx < subgraph.tensorsLength(); tensorIdx++) {
+    let tensor = subgraph.tensors(tensorIdx);
+
+    let hashnode: Node = {name: tensor!.name()!, type: tensor!.shapeArray()!};
```

Comment:
Can I ask you why `type` is used for indicating `shape`?

Reply:
![image](https://user-images.githubusercontent.com/47562093/135552932-fefc8467-3d1a-4f39-9538-b59774b076a4.png)

The variable refers to the arrangement in the photo.

Reply:
Hmm... `type` means `float32`, `int8`, etc.
That is shape...!

Reply:
There are two types of data displayed in the type variable.
It is a type such as `float32`, `int8` and a `shapeArray` expressed together with the type.
Here we do not yet see a type such as `float32`. Instead, it denotes `shapeArray`.
If you look at the places colored with a highlighter, there is a `shapeArray`, so I put it in the type variable.

Reply:
Then type such as `float32, `int8` will be displayed later? :)

Reply:
Yes. Small parts not added now will be implemented later.

Reply:
Then please add TODO comment about it in the source code..!